qa/workunits/cephadm/test_iscsi_etc_hosts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# checks if the container and host's /etc/hosts files match
+# Necessary to avoid potential bugs caused by podman making
+# edits to /etc/hosts file in the container
+# exits with code 1 if host and iscsi container /etc/hosts do no match
+
+set -e
+set -x
```

Comment:
These are usually combined, leaving a blank line after to make the settings stand out:
```suggestion
set -ex

```